### ISSUE TITLE
Gamma's Froggy Hunt cutscene

### ISSUE THREAD

**Issue by PiKeyAr** [SonicFreak94/sadx-dc-lighting#51]
It looks like SA1 has a lighting change in the cutscene where Eggman talks to the Egg Carrier crew.
![dafmv6fu0aa1w-2](https://cloud.githubusercontent.com/assets/23312549/26248746/89885ac6-3cde-11e7-922a-f0cf49ec09e0.jpg)
Unless there is some magic involving SL files, I think the base palette is being temporarily overwritten with dark gray until the moment the lights turn on, or the base palette's brightness is lowered.

Recreating this effect will also help #26.

**Comment by PiKeyAr** [comment]
Here's a [savegame](https://dcmods.unreliable.network/owncloud/data/PiKeyAr/files/SonicDX02.snc) that starts Gamma's story right on the cutscene.

**Comment by PiKeyAr** [comment]
I implemented it in DC_EggCarrier using the following:

sub_6F4577 loads an object (ObjectMaster at dword_3C85138) that is supposed to turn off the lights.
sub_6F4620 destroys that object right before Eggman starts speaking.

I loaded a modified PL file that has a grey diffuse palette (FF646464) and hooked the above functions to blend into this palette when the lights are off, and disable blending when the lights turn on.
